src/vcs/git/repo.rs

```diff
@@ -117,6 +117,24 @@ impl<'a> RepositoryRef<'a> {
             })
     }
 
+    /// List the namespaces within a repository, filtering out ones that do not
+    /// parse correctly.
+    ///
+    /// # Errors
+    ///
+    /// * [`Error::Git`]
+    pub fn list_namespaces(&self) -> Result<Vec<Namespace>, Error> {
```

Comment:
Any reason we shouldn't call both these functions simply `namespaces()`?

Reply:
The rest of the code uses prefixes as well: `list_branches`, `list_tags`, etc.